### ISSUE TITLE
Visualising and communicating uncertainty

### ISSUE THREAD

**Issue by ghost** [ropensci/auunconf#13]
With the launch of the National Innovation and Science Agenda late last year, there is a strong focus to develop new and innovative ideas to help government and industry solve real world problems. While this sounds easy enough, deriving solutions for real world problems can be quite challenging, particularly if we are trying to model complex system processes and reconcile this with actual measurements. 

I think the one thing that is lacking is the interpretation of these complex modelling frameworks and more importantly, the communication of uncertainty and what it means to the end user. More often than not, we present the outputs from our “model” in a paper/report that does not have any uptake or stakeholder engagement. How can we change this? How can our models have impact on society?

I would be interested in engaging in a discussion around this topic to determine if there is a general tool or suite of tools that can be developed to assist end users/stakeholders with the interpretation of the outputs from complex models and translate the outputs into impactful decisions.  I recognise that this may be problem specific but I think the topic of communicating and visualising uncertainty is an important one that is not done well.

To help facilitate this discussion I have an example from the Great Barrier Reef where interest lies with the prediction of sediment loads. We have developed a Bayesian Hierarchical model for blending modelled output with measurements to provide a spatial map of sediment predictions through time with uncertainties. We are currently exploring ways to present these results to end users through a tool to assist with decision making. It occurred to me that the problem of communicating and visualising uncertainty and making decisions with confidence may not be unique and may benefit other modellers and end users wishing to understand the model that has been delivered.

**Comment by jesse-jesse** [comment]
Hey Petra, 

This is a great problem!! And the topic of my current research masters, so I would love to explore this further. 

I am currently looking at communicating uncertainty to non-expert audiences within disease maps. But would love to have a chat about the Barrier reef context.

**Comment by ghost** [comment]
Great Jess.  You should bring along your problem as well to discuss to see if there are common threads. I'm sure there will be. The communication side of things is a difficult one in my experience.

**Comment by tslumley** [comment]
I'm interested in this.  Maybe it would be best to implement simple versions of a few ideas so they can be tested with the target audience, who are probably not the sort of people who don't go to stats software unconferences.

**Comment by ghost** [comment]
Yes, I agree.  I just proposed the GBR work to hopefully generate some interest and discussion. Definitely biting off smaller chunks to showcase the idea would be a great way to start. Looking forward to discussing this in a couple of weeks.

**Comment by jesse-jesse** [comment]
One of the difficulties in visualising uncertainty is that there are so many different types of uncertainty. We could explore a visualisation technique for a specific uncertainty  measure - like the credible interval. Or maybe an uncertainty audit took that can help communicators identify which uncertainties are most important to their application.

**Comment by jesse-jesse** [comment]
I just realised how ridiculous my profile picture is in the context of this discussion. I will have to update that .. ..

**Comment by jesse-jesse** [comment]
I also think that interactivity can be a useful channel for including uncertainty. Often the visual channels (colour, size,shape etc) are taken up by other variables.

**Comment by jesse-jesse** [comment]
Was great to have an opportunity to chat about this sticky challenge. Although we didn't get that far in building anything I'm keen to keep working on this one.. Hopefully get a change to have a play with the visualisation in the next few weeks.

**Comment by ghost** [comment]
Yes it is a tricky one. I'm working on aspects of this for my specific problem. Maybe we can touch base sometime.

**Comment by jesse-jesse** [comment]
sounds good..
